### PR TITLE
Feature/setup routes

### PR DESCRIPTION
**Pull request by Oliver-was-taken**
### What?
Setup routing

### Why?
Switch between creation/game

### How?
react-router-dom

### CODE REVIEW

**Review by kaibria — APPROVED** [review]
Good idea

**Review by tamer545 — APPROVED** [review]
Good creation of routes

File: fabbwled-frontend/src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 function App() {
-  return (
-    <Routes>
-      <Route path="/create" />
-      <Route path="/game" />
-    </Routes>
-  );
+    return (
+        <Routes>
+            <Route path="/create" element={<Create/>}/>
```
Comment (Nilstrieb) [inline fabbwled-frontend/src/App.tsx:8]: 👍 good to see automated formatting

**Comment by miolmi** [comment]
vely nice

**Review by Nilstrieb — APPROVED** [review]

**Review by MagischerWolf05 — APPROVED** [review]

**Review by nnikolajj — APPROVED** [review]

**Review by corsin-lehre — APPROVED** [review]
I am amazed about how this pr saved me from unaliving myself

**Review by JJoeeyy — CHANGES_REQUESTED** [review]
Nice work, thank you oliver

**Review by JJoeeyy — APPROVED** [review]
nice work, thank you oliver

**Pull request merged**